feedback/api/views.py

```diff
@@ -0,0 +1,85 @@
+from django.http import HttpResponse, JsonResponse
+
+from rest_framework.decorators import (
+    APIView,
+    api_view,
+    authentication_classes,
+    permission_classes,
+)
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+
+from feedback.models import Reaction, Comment
+from news.models import News, Subscriber
+from news.utils.authentication import get_user_By_token
+
+from response.rest import (
+    OkResponse,
+    NotFoundResponse,
+    CreateUserSuccessResponse,
+    CreateUserErrorResponse,
+    LoginSuccessResponse,
+    LoginErrorResponse,
+    NotAuthorizedResponse,
+)
+
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the news index.")
+
+
+class ReactToNews(APIView):
+    def post(self, request, pk):
+        try:
+            news = News.objects.get(token=pk)
+            react = request.data.get('reaction')
```

Comment:
Also it's better to use the same file structure that we have in news app. Move the logic to the "service". Move queries to a "utils" file related to the app. Only get request data and create responses based on returned data from service in api directory.